<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>zef IDE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>zefLogo.png</normaloff>zefLogo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0,0,0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Save as</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>650</y>
      <width>971</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13,13,13);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>41</width>
      <height>838</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(184, 59, 47);
background-color: rgb(35, 37, 39);
font:10pt &quot;Verdana&quot;;
margin-top: 0px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>670</y>
      <width>971</width>
      <height>228</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13,13,13);
border-style:none;
border-width:0px;
color: rgb(66, 150, 158);
padding-left:15px;
padding-top:5px;
padding-bottom:10px;

 </string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="cursorWidth">
     <number>1</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="MyGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>2000</width>
      <height>971</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>591</width>
      <height>838</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(29, 31, 33);
font: 10pt &quot;Verdana&quot;;
color:rgb(100,100,100);
border-style:none;
padding-left:5px;
padding-bottom:5px;
</string>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="tabStopDistance">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>641</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(184, 59, 47);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>                 Editor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(184, 59, 47);
background-color: rgb(13,13,13);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>1601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>30</y>
      <width>1601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(184, 59, 47);
background-color: rgb(21, 21, 21);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>961</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(184, 59, 47);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Canvas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>40</y>
      <width>1711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 37, 39);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>650</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 210, 210);
background-color: rgb(184, 59, 47);
border-style:none;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>      Console   </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 37, 39);
border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>33</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Toggle Live</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>8</width>
      <height>8</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(66, 150, 158);
background-color: rgb(29, 31, 33);</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>649</y>
      <width>971</width>
      <height>6</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
background-color: rgb(184, 59, 47);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>897</y>
      <width>1621</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
color: rgb(184, 59, 47);
background-color: rgb(20,20,20);
font:8pt &quot;Verdana&quot;;
margin-top: 0px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>label_5</zorder>
   <zorder>line_2</zorder>
   <zorder>label_6</zorder>
   <zorder>toolButton_5</zorder>
   <zorder>toolButton</zorder>
   <zorder>toolButton_9</zorder>
   <zorder>toolButton_6</zorder>
   <zorder>toolButton_7</zorder>
   <zorder>toolButton_4</zorder>
   <zorder>widget</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>textBrowser_3</zorder>
   <zorder>line</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>label_10</zorder>
   <zorder>toolButton_10</zorder>
   <zorder>label_11</zorder>
   <zorder>toolButton_11</zorder>
   <zorder>label_9</zorder>
   <zorder>label</zorder>
   <zorder>textBrowser_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">zefopengl.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
